entrypoint.sh

```diff
@@ -25,10 +34,11 @@ if [ -n "$local_keymap" ]; then
 fi
 
 qmk config user.qmk_home=/opt/qmk_firmware
-qmk compile -kb "$keyboard" -km "$keymap"
+echo Controller - "$controller"
```

Comment:
This message may be confusing when we're not specifying a controller to be converted to. If there is an easy way to grep controller information through the QMK CLI, we could do that to determine which one is being used, but otherwise I think we should only "say" something when the input has been given.

```suggestion
if [ -n "$controller" ]; then
  echo "Using CONVERT_TO=$controller with qmk..."
fi
```

Reply:
This was a leftover testing string, i have removed it in https://github.com/d3adb5/build-qmk-fw/pull/4/commits/00bfad465d68412d568143a9ba7c0a9a355a1aa9